hazelcast/src/main/java/com/hazelcast/cache/impl/nearcache/impl/store/AbstractNearCacheRecordStore.java

```diff
@@ -37,8 +37,8 @@
 import com.hazelcast.util.Clock;
 import com.hazelcast.util.ExceptionUtil;
 
-import static com.hazelcast.internal.memory.GlobalMemoryAccessorRegistry.MEM;
-import static com.hazelcast.internal.memory.GlobalMemoryAccessorRegistry.MEM_AVAILABLE;
+import static com.hazelcast.spi.memory.GlobalMemoryAccessorRegistry.MEM;
```

Comment:
Do we want the memory package to become an SPI? SPI has more ritual than internal.